From: Will Thompson <wjt@endlessm.com>
Date: Wed, 22 Aug 2018 16:51:08 +0100
Subject: transaction: fix signal parameter documentation

I guess these predate the creation of FlatpakTransactionOperation.

Closes: #1995
Approved by: alexlarsson

(cherry picked from commit 04846e24b2903e2d46c0bd3daeb7a04e7b5ddc23)
---
 common/flatpak-transaction.c | 18 +++++-------------
 1 file changed, 5 insertions(+), 13 deletions(-)

diff --git a/common/flatpak-transaction.c b/common/flatpak-transaction.c
index 3e9d902..b391ca8 100644
--- a/common/flatpak-transaction.c
+++ b/common/flatpak-transaction.c
@@ -654,11 +654,8 @@ flatpak_transaction_class_init (FlatpakTransactionClass *klass)
   /**
    * FlatpakTransaction::new-operation:
    * @object: A #FlatpakTransaction
-   * @ref: The ref the operation will be working on
-   * @remote: The ref the operation will be working on
-   * @bundle: The bundle path (or %NULL)
-   * @operation_type: A #FlatpakTransactionOperationType specifying operation type
-   * @progress: A #FlatpakTransactionProgress
+   * @operation: The new #FlatpakTransactionOperation
+   * @progress: A #FlatpakTransactionProgress for @operation
    */
   signals[NEW_OPERATION] =
     g_signal_new ("new-operation",
@@ -672,11 +669,9 @@ flatpak_transaction_class_init (FlatpakTransactionClass *klass)
   /**
    * FlatpakTransaction::operation-error:
    * @object: A #FlatpakTransaction
-   * @ref: The ref the operation was working on
-   * @remote: The remote
-   * @operation_type: A #FlatpakTransactionOperationType specifying operation type
+   * @operation: The #FlatpakTransactionOperation which failed
    * @error: A #GError
-   * @details: A #FlatpakTransactionErrorDetails with Details about the error
+   * @details: A #FlatpakTransactionErrorDetails with details about the error
    *
    * Returns: the %TRUE to contine transaction, %FALSE to stop
    */
@@ -692,10 +687,7 @@ flatpak_transaction_class_init (FlatpakTransactionClass *klass)
   /**
    * FlatpakTransaction::operation-done:
    * @object: A #FlatpakTransaction
-   * @ref: The ref the operation was working on
-   * @remote: The remote
-   * @operation_type: A #FlatpakTransactionOperationType specifying operation type
-   * @commit: The new commit checksum
+   * @operation: The #FlatpakTransactionOperation which finished
    * @result: A #FlatpakTransactionResult giving details about the result
    *
    */
